solr/core/src/test/org/apache/solr/TestSolrCoreProperties.java

```diff
@@ -79,11 +79,10 @@ public static void beforeTest() throws Exception {
     if (System.getProperty("solr.data.dir") == null) {
       nodeProperties.setProperty("solr.data.dir", createTempDir().toFile().getCanonicalPath());
     }
-    jetty =
-        new JettySolrRunner(homeDir.getAbsolutePath(), nodeProperties, buildJettyConfig("/solr"));
 
-    jetty.start();
-    port = jetty.getLocalPort();
+    new JettySolrRunner(homeDir.getAbsolutePath(), nodeProperties, buildJettyConfig("/solr"));
```

Comment:
does this guy need to be set to something?

Reply:
Right! He was supposed to do this `solrClientTestRule.startSolr(Path.of(homeDir.getAbsolutePath()), nodeProperties, buildJettyConfig("/solr"));`